### DISCUSSION TITLE
WinUI licence

### DISCUSSION THREAD

**Discussion by AdrasteonDev** [microsoft/WindowsAppSDK#1007] [Q&A]
I am a bit confused about the WinUI license.

The [WinUI](https://github.com/microsoft/microsoft-ui-xaml) repository is using the MIT license, and this repository is using CC BY 4.0. But the [Nuget](https://www.nuget.org/packages/Microsoft.ProjectReunion.WinUI/0.8.0/License) [packages](https://www.nuget.org/packages/Microsoft.UI.Xaml/2.6.0/License) are using "MICROSOFT SOFTWARE LICENSE TERMS".

IANAL, but it seems that this license is not compatible with open source licenses like MIT or Apache. It requires to accept data collection, and 2.c.ii seems to prohibit  the distribution of the code.

Is there a possibility that the license will change one day ? If not, do you know if it is legal / possible to release an open source application (with an Apache or MIT license) using these Nuget packages ?

**Comment by fredemmott** [comment]
@gabbybilka is there any progress on open sourcing? If this isn't imminent, please prioritize:

- making the licensing information consistent
- the creation and registration of a suitable [SPDX license exemption](https://spdx.org/licenses/exceptions-index.html) - so, for example, open source projects can be licensed as 'GPL, but with an exception to allow linking and distribution with the windows app sdk'

**Reply by gabbybilka** [reply]
@bpulliam @MikeHillberg
